thirdeye/thirdeye-frontend/app/pods/components/filter-bar/template.hbs

```diff
@@ -0,0 +1,28 @@
+{{!-- TODO: Create separate styling for filter-bar component --}}
+<div class="entity-filter filter-bar">
+
+  {{#each config as |block|}}
+    <section class="entity-filter__group">
+      <h5 class="entity-filter__group-title">{{block.header}}
+        <i class="glyphicon"></i>
+      </h5>
+      <ul class="entity-filter__group-list {{if block.isHidden " entity-filter__group-list--hidden "}}">
```

Comment:
no need for space before and after the entity-filter__group-list--hidden class name `"entity-filter__group-list--hidden"`